.gitignore

```diff
@@ -0,0 +1,5 @@
+webapp/node_modules
+webapp/.cache
+webapp/dist
+.idea
+target/
```

Comment:
nit: consider sorting this list [in ASCII sort order](https://en.wikipedia.org/wiki/ASCII#Printable_characters) to ease maintenance (`.` comes before letters); along with that, add [a comment](https://stackoverflow.com/questions/8865848/comments-in-gitignore) (line starting with `#`) at the beginning along the lines of "Please keep this list in ASCII sort order".

Reply:
Updated here 4454777 and 95786b2